### PR TITLE
Implement a breadcrumb module

### PR DESCRIPTION
**Pull request by Hackwar**
This will implement the breadcrumb as a separate module.

Right now this is just a draft. I'm still working on it.

### CODE REVIEW

File: administrator/modules/mod_breadcrumbs/mod_breadcrumbs.php [hunk 1]
```diff
@@ -19,6 +19,11 @@
 
 // Get the breadcrumb items
 /** @var \Joomla\CMS\Pathway\AdministratorPathway $pathway */
-$this->breadcrumbs['breadcrumbs'] = $app->getPathway()->getPathWay();
+$pathway = $app->getPathway();
+$items = $pathway->getPathWay();
+$items[0]->name = JText::_('MOD_MENU_CONTROL_PANEL');
```
Comment (brianteeman) [inline administrator/modules/mod_breadcrumbs/mod_breadcrumbs.php]: Please use the namespaced Text::_

File: administrator/language/en-GB/en-GB.mod_breadcrumbs.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+MOD_BREADCRUMBS="Administrator Breadcrumbs"
+MOD_BREADCRUMBS_XML_DESCRIPTION="Display a breadcrumb in the administrator backend."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_breadcrumbs.ini]:
```suggestion
MOD_BREADCRUMBS_XML_DESCRIPTION="Display an administrator breadcrumb module."
```

File: administrator/language/en-GB/en-GB.mod_breadcrumbs.sys.ini [hunk 3]
```diff
@@ -0,0 +1,7 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+MOD_BREADCRUMBS="Administrator Breadcrumbs"
+MOD_BREADCRUMBS_XML_DESCRIPTION="Display a breadcrumb in the administrator backend."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_breadcrumbs.sys.ini]:
```suggestion
MOD_BREADCRUMBS_XML_DESCRIPTION="Display an administrator breadcrumb module."
```

File: administrator/modules/mod_breadcrumbs/mod_breadcrumbs.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @package     Joomla.Site
```
Comment (brianteeman) [inline administrator/modules/mod_breadcrumbs/mod_breadcrumbs.php]:
```suggestion
 * @package     Joomla.Administrator
```

File: administrator/modules/mod_breadcrumbs/mod_breadcrumbs.xml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="module" version="4.0" client="administrator" method="upgrade">
+	<name>mod_breadcrumbs</name>
+	<author>Joomla! Project</author>
+	<creationDate>May 2019</creationDate>
+	<copyright>Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>3.0.0</version>
```
Comment (brianteeman) [inline administrator/modules/mod_breadcrumbs/mod_breadcrumbs.xml]:
```suggestion
	<version>4.0.0</version>
```

File: administrator/modules/mod_breadcrumbs/tmpl/default.php [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @package     Joomla.Site
```
Comment (brianteeman) [inline administrator/modules/mod_breadcrumbs/tmpl/default.php]:
```suggestion
 * @package     Joomla.Administrator
```

**Comment by Hackwar** [comment]
Unfortunately the whole concept of breadcrumbs in the backend can't really work when the backend has no concept of a menu system/hierarchy. This together with the questionable benefit for the users brings me to the point that I'm rejecting my own PR here.

**Pull request closed without merging**